pageserve/app.py

```diff
@@ -76,36 +76,36 @@ def parsed_posts(posts):
 
 def get_events():
     """Gets all sub-events from events service."""
-    # TODO(cmei4444): integrate with events service to pull event info from
-    # database
-    events = [{'event_id': '1',
-               'name': 'concert 1',
-               'description': 'listen to fun music here!',
-               'author': 'admin',
-               'created_at': '7-9-2019',
-               'event_time': '7-10-2019',
-               },
-              {'event_id': '2',
-               'name': 'concert 2',
-               'description': 'listen to fun music here!',
-               'author': 'admin',
-               'created_at': '7-9-2019',
-               'event_time': '7-12-2019',
-               }]
-    return parsed_events(events)
-
-
-def parsed_events(events):
-    # TODO(cmei4444): implement parsing on events pulled from events service in
-    # a format for web display
-    return events
+    url = os.environ.get("EVENTS_ENDPOINT")
+    r = requests.get(url, params={})
+    if r.status_code == 200:
+        return parsed_events(r.json())
+    else:
+        # TODO(cmei4444): handle error in a way that doesn't break page display
+        return "Error in getting events"
+
+
+def parsed_events(events_dict):
```

Comment:
Name this parse_events or get_parsed_events so that its an active description of what the function does.

Reply:
Done in f84f9a0